web/src/navbar_alerts.ts

```diff
@@ -162,6 +160,36 @@ export function should_offer_to_update_timezone(): boolean {
     );
 }
 
+const DESKTOP_NOTIFICATIONS_BANNER: AlertBanner = {
+    process: "desktop-notifications",
+    intent: "brand",
+    label: new Handlebars.SafeString(
+        $t_html({
+            defaultMessage:
+                "Zulip needs your permission to enable desktop notifications for important messages.",
+        }),
+    ),
+    buttons: [
+        {
+            type: "primary",
+            label: "Enable notifications",
```

Comment:
Labels not tagged for translation don't work.

Reply:
Fixed. My bad, it was really obvious but I somehow overlooked it.